Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -425,12 +440,20 @@ public static Furniture PlaceInstance(Furniture proto, Tile tile)
         return obj;
     }
 
+    public void EveryFrameUpdate(float deltaTime)
```

Comment:
We should add docs here and update the docs in FixedFrameUpdate, so that is clear why there are 2 update methods.